inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/animation/ItemExpandAnimator.kt

```diff
@@ -42,5 +42,11 @@ abstract class ItemExpandAnimator {
      */
     @JvmStatic
     fun split() = SplitExpandAnimator()
+
+    /**
+     * See [ScaleExpandAnimator].
+     */
+    @JvmStatic
+    fun scale(allowScaling: Boolean = true) = ScaleExpandAnimator(allowScaling)
```

Comment:
I think `allowScaling` param can be named better. It sounds confusing that a function named `scale()` has a param called `allowScaling`. Maybe `scaleBackground`, `scaleListItem`, `pushBackground` or something else?

Reply:
Done